### ISSUE TITLE
Image Size Analysis: Exclude External Images

### ISSUE THREAD

**Issue by bobmatyas** [Automattic/jetpack#34199]
### Impacted plugin

Boost

### Quick summary

Image Size Analysis makes recommendations for external images included via widgets, but the user doesn't control over the size of those images.

This results in a number of recommendations that can't be acted on. Additionally, Site Accelerator doesn't resize the images.

### Steps to reproduce

1. Use a widget (Jetpack Display Remote Posts) to import blog posts from a third-party site
2. Run Image Size Analysis
3. Observe numerous recommendations for image adjustments that can't be made

### A clear and concise description of what you expected to happen.

I would expect that we either exclude external images or indicate that they can't be optimized because they are external images.

I also would have expected Site Accelerator to present these at appropriately sized dimensions	

### What actually happened

External Images Being Loaded:

![Annotation on 2023-11-18 at 09-22-37](https://github.com/Automattic/jetpack/assets/45246438/d4fd8196-329f-4fe6-8449-d399903800d0)

Results in Image Size Analysis:

![Annotation on 2023-11-18 at 09-24-48](https://github.com/Automattic/jetpack/assets/45246438/60957911-9eb1-48d5-bc45-9876c0067490)

### Impact

Some (< 50%)

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Self-hosted

**Comment by thingalon** [comment]
@pyronaur  - I would love your thoughts on this. It seems like a quick thing to fix but I want to double-check that the recommended solution is the right approach here first.

**Comment by pyronaur** [comment]
We can't really differentiate between external images that can or can't be optimized. We probably can't run any auto-fixers or Jetpack CDN to fix them, but we should still let the user know that they're wrecking the performance.

Off the top of my head - a common situation here might be where the user is using some CDN plugin that hosts the images on a  different domain.

Sometimes they might not have control, but sometimes they might. Instead of removing, I think this is a point we have to consider through the design of the dashboard.

**Comment by bobmatyas** [comment]
Yes, I think handling via better design in the dashboard would be a good fix here. I only considered direct imports from third-party sites, not the CDN use case.

**Comment by kraftbj** [comment]
Closing this issue as the Image Size Analysis (ISA) feature has been removed from Boost.
